flink-connectors/flink-connector-kinesis/src/test/java/org/apache/flink/streaming/connectors/kinesis/FlinkKinesisConsumerMigrationTest.java

```diff
@@ -95,18 +94,12 @@ public class FlinkKinesisConsumerMigrationTest {
 
     @Parameterized.Parameters(name = "Migration Savepoint: {0}")
     public static Collection<FlinkVersion> parameters() {
-        return Arrays.asList(
-                FlinkVersion.v1_3,
-                FlinkVersion.v1_4,
-                FlinkVersion.v1_7,
-                FlinkVersion.v1_8,
-                FlinkVersion.v1_9,
-                FlinkVersion.v1_10,
-                FlinkVersion.v1_11,
-                FlinkVersion.v1_12,
-                FlinkVersion.v1_13,
-                FlinkVersion.v1_14,
-                FlinkVersion.v1_15);
+        final Collection<FlinkVersion> supportedVersions =
+                FlinkVersion.rangeOf(
+                        FlinkVersion.v1_3, FlinkVersion.mostRecentlyPublishedBaseMinorVersion());
+        // TODO: it's unclear why we skipped 1.6
+        supportedVersions.remove(FlinkVersion.v1_6);
```

Comment:
```suggestion
        supportedVersions.remove(FlinkVersion.v1_5);
        supportedVersions.remove(FlinkVersion.v1_6);
```
based on the list it seems `v1_5` should also be skipped